.github/workflows/publish.yml

```diff
@@ -164,6 +164,12 @@ jobs:
           apt install -y wget curl build-essential
       - name: Print GLIBC version
         run: ldd --version
+      - name: Verify GLIBC version
+        run: |
+          set -eux -o pipefail
+
+          LDD_VERSION=$(ldd --version | head -n 1 | sed -E 's/.*GLIBC ([0-9]+[.][0-9]+).*/\1/')
+          [ $LDD_VERSION == "2.31" ]
```

Comment:
can we be even simpler?

```
ldd --version | grep 'GLIBC 2.31'
```

should do the trick

Reply:
done.